### PR TITLE
feat:create cooperative folder

### PR DESCRIPTION
**Pull request by Eddahkarambu**
# [32 ] Social Impact Cooperative Section

This is a section that describes the organizations' common objectives

# Changes
The component displays by reusing the banners feature from  #31

# Desktop preview
cooperative1

# Mobile preview
cooperative2

# Contributors
# Code was written by @Essyuge , @JohnstoneDev & I .

### CODE REVIEW

**Comment by Eddahkarambu** [comment]
@LemmyMwaura kindly review

File: libs/pages/elewa/home/src/lib/components/cooperative-section/cooperative-section.component.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<elewa-group-elewa-group-image-and-text-banner 
+  [paragraphTexts]="paragraph_text "
+  imageURL="{{ image_src }}"
```
Comment (LemmyMwaura) [inline libs/pages/elewa/home/src/lib/components/cooperative-section/cooperative-section.component.html:3]: This is not 'binded' to the image source, you should also pass the title

**Review by LemmyMwaura — COMMENTED** [review]
This issue has already been closed but good job so far.

**Comment by IanOdhiambo9** [comment]
This issue has been closed

**Pull request closed without merging**